.travis.yml

```diff
@@ -28,6 +28,7 @@ matrix:
 
 install:
   - travis_retry pip install tox
+  - pip install git+https://github.com/sydoluciani/deform.git@deform_replacing_help_block
```

Comment:
Let's leave this line out of this PR. It needs to be removed eventually before I merge it. We know that there is a chicken-and-egg situation. Also `master` is not a release branch, so it is OK to break functional tests in deformdemo temporarily as long as it is communicated, which you have done.

Typically I pull down the changes in the PRs in both Deform and deformdemo and verify that tests pass locally if I have any doubt.

Reply:
Done, removed it.

That line didn't work. tox using setup.py, and deform in setup.py is being installed from pip repo.
I was trying to implement dependency_links in setup.py but noticed it is [depricated](https://setuptools.readthedocs.io/en/latest/setuptools.html)

Please check, merge and push the Deform to pip repository and update setup.py to include newer version of [Deform](https://github.com/Pylons/deformdemo/blob/master/setup.py#L34), otherwise Deformdemo would fail even after merging Deform pull request.

after the success on both Deform and Deformdemo merge, I will submit another pull request to fix the input field color.

Reply:
I'm going to hold off on merging this PR, because I think there are more changes needed in Deform. Let's discuss in that issue and agree on a plan forward to save unnecessary work.